packages/@azure/servicebus/data-plane/examples/javascript/gettingStarted/loginWithAzureAccount.js

```diff
@@ -0,0 +1,40 @@
+/*
+  This sample demonstrates how to create a namespace using AAD token credentials
+  obtained from signing in through your Azure account.
+
+  Setup :
+    Please ensure that your Azure Service Bus resource is in US East, US East 2, or West Europe
+    region. AAD Role Based Access Control is not supported in other regions yet.
+
+    Assign "owner" role to the registered application
```

Comment:
Assign "owner" role to the user account or registered application?

Reply:
Oops